packages/mosaic/tree/tree-selection.component.ts

```diff
@@ -107,17 +103,16 @@ export class McTreeSelection extends CdkTree<McTreeOption>
 
     @Output() readonly selectionChange = new EventEmitter<McTreeSelectionChange>();
 
-    @Input()
+    multipleMode?: MultipleMode;
+
     get multiple(): boolean {
-        return this._multiple;
+        return !!this.multipleMode;
     }
 
     set multiple(value: boolean) {
```

Comment:
you have public `multipleMode` and setter `multiple(value: boolean)` for it, why and what do you solve by this ?

Reply:
it is two ways of mode setting - `multipleMode`  is direct setting of multiple mode and `multiple` is setting multiple mode in common without knowledge about default `multipleMode` nad means  set multiple mode by default or off it